packages/vuetify/src/composables/border-radius.ts

```diff
@@ -0,0 +1,61 @@
+// Utilities
+import { computed } from 'vue'
+import propsFactory from '@/util/propsFactory'
+
+// Types
+import type { PropType } from 'vue'
+
+export type BorderRadiusProp = boolean | 0 | '0' | '' | 'xs' | 'sm' | 'lg' | 'xl' | 'circle' | 'pill' | 'shaped' | 'tile' | null
```

Comment:
All types in one type declaration, gr8... wait!

There's `pill`, `shaped`, `tile`, and I don't see them in the tests above :(